docs/usability/using_sig_db_tool.md

```diff
@@ -0,0 +1,33 @@
+# UEFI Secure Boot Database Inspection Tool
+
+A simple command-line interface to the UEFI Secure Boot Database Library focused on database inspection
+
+## Usage info
+
+``` cmd
+usage: sig_db_tool [-h] [--compact] {dump,get_dupes,get_canonical} ...
```

Comment:
why does usage not show the positional argument of the actual bin file?  

What is the bin file?  Given how many layers of stuff can be in the Auth variable especially when you want to write it...i think being clear what is the bin file.

Reply:
the content from https://github.com/tianocore/edk2-pytool-extensions/pull/215/files#diff-ce00158dd19f56a7155523f2e5ce0fa6R26 would be good

Reply:
That is how Python's argparse help works.  It won't tell you the arguments for the subparsers until you request help for each subparser.  If you scroll down in the usage, there is 1 example of each.  However, I will add further examples here in the readme.

Reply:
Latest commit below has usage for each action, and a real example for each action